env0/resource_configuration_variable.go

```diff
@@ -205,8 +199,8 @@ func resourceConfigurationVariableRead(ctx context.Context, d *schema.ResourceDa
 					d.Set("enum", variable.Schema.Enum)
 				}
 
-				if variable.Schema.Format == client.HCL {
-					d.Set("format", string(client.HCL))
+				if variable.Schema.Format != "" {
+					d.Set("format", variable.Schema.Format)
 				}
```

Comment:
Shouldn't this whole logic be duplicated to environment resource's configuration variables as well?

Reply:
@RLRabinowitz looks like it's already there...
https://github.com/env0/terraform-provider-env0/blob/2ad99bb9fc7cfd0eef617f24e544bfa8298906aa/env0/resource_environment.go#L579-L583